find_changed_files.sh

```diff
@@ -1,28 +1,38 @@
 #!/bin/bash
 # Find changed files in a build either via pull request API or compare API
 set -x
+INCLUDE_REMOVED=$1
+
+if [[ $INCLUDE_REMOVED == true  ]]
+then
+    EXCLUDED_STATUS=""
+else
+    EXCLUDED_STATUS="removed"
+fi
+
 cat $GITHUB_EVENT_PATH
 COMPARE=$(jq '.compare' $GITHUB_EVENT_PATH)
 PR=$(jq -r '.pull_request._links.self.href' $GITHUB_EVENT_PATH)
 echo "-----"
```

Comment:
While we're here, can we also clean up these echo statements since they're redundant?

Reply:
which statements?